tutorials/export-audit-trail-to-datadog/index.mdx

```diff
@@ -0,0 +1,174 @@
+
+## Building the OpenTelemetry collector
+
+1. Open a terminal and check that the `ocb` binary is ready to be used. The output of the `help` command should display, meaning the `ocb` binary is ready to be used.
+
+    ```
+    ./ocb help
+    ```
+
+2. Create a manifest file named `builder-config.yaml` and paste the following content into it. This file is used to defines code generation, the compile process, and the components to include in your Collector’s distribution.
+
+    ```yaml
+    dist:
+      name: otelcol-audit-trail
+      description: OpenTelemetry Collector for Audit Trail
+      output_path: ./otelcol-audit-trail
+    
+    exporters:
+      - gomod:
+          github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter v0.118.0
+    
+    processors:
+      - gomod:
+          go.opentelemetry.io/collector/processor/batchprocessor v0.118.0
+    
+    receivers:
+      - gomod:
+          github.com/scaleway/opentelemetry-collector-scaleway/receiver/scwaudittrail v0.1.0
+    
+    providers:
+      - gomod: go.opentelemetry.io/collector/confmap/provider/envprovider v1.24.0
+      - gomod: go.opentelemetry.io/collector/confmap/provider/fileprovider v1.24.0
+      - gomod: go.opentelemetry.io/collector/confmap/provider/httpprovider v1.24.0
+      - gomod: go.opentelemetry.io/collector/confmap/provider/httpsprovider v1.24.0
+      - gomod: go.opentelemetry.io/collector/confmap/provider/yamlprovider v1.24.0
+    ```
+
+3. Run the following command to build the Collector. `GOOS` and `GOARCH` are needed in the command as the target deployment is Linux with an AMD64 CPU.
+
+    ```
+    GOOS=linux GOARCH=amd64 ./ocb --config builder-config.yaml
+    ```
+
+You now have a new folder named `otelcol-audit-trail/` with the binary `otelcol-audit-trail` compiled inside.
+
+## Deploying the Collector
+
+1. Run the following command to upload the Collector binary to your Instance. Make sure that you replace `<INSTANCE_IP_ADDRESS>` with the IP address of your Instance.
+
+    ```
+    scp otelcol-audit-trail/otelcol-audit-trail root@<INSTANCE_IP_ADDRESS>:/usr/local/bin/
+    ```
+
+2. Connect to your Instance via SSH:
+
+    ```
+    ssh root@<INSTANCE_IP_ADDRESS>
+    ```
+
+## Configure the Collector
+
+Create a file named `/etc/opentelemetry-collector/config.yaml` and paste the following content into it. This file is the configuration our custom Collector will run. 
+
+    ```yaml
+    receivers:
+      scwaudittrail:
+        access_key: <SCW_ACCESS_KEY>
+        secret_key: <SCW_SECRET_KEY>
+        organization_id: <SCW_DEFAULT_ORGANIZATION_ID>
+        region: <SCW_DEFAULT_REGION>
+    
+    processors:
+      batch:
+        send_batch_max_size: 1000
+        send_batch_size: 100
+        timeout: 10s
+    
+    exporters:
+      datadog:
+        idle_conn_timeout: 10s
+        api:
+          key: <DD_API_SECRET_KEY>
```

Comment:
```suggestion
          key: <DD_API_KEY>
```
When using DataDog API keys, usually it is exposed as the environment variable `DD_API_KEY`, that's why I named it that way in the doc